src/index.html

```diff
@@ -9,10 +9,58 @@
     <title>Stars</title>
     <link
       rel="stylesheet"
-      href="./style.css"
+      href="./styles/style.css"
     />
```

Comment:
The attributes of the link tag should also start on a new line, each with a 2-space indentation related to the tag, to keep the attributes correctly formatted.